src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/dynamiclists/DynamicJudgements.java

```diff
@@ -39,24 +40,57 @@ private static void populateDynamicJudgements(CaseData caseData, List<DynamicVal
                 var judgementCollection = caseData.getJudgementCollection();
                 for (JudgementTypeItem judgementTypeItem : judgementCollection) {
                     dynamicHearingDate(caseData, hearingDynamicList, judgementTypeItem.getValue());
+                    dynamicReconsiderations(caseData, parties, judgementTypeItem.getValue());
                 }
             } else {
-                createDynamicJudgment(caseData, hearingDynamicList);
+                createDynamicJudgment(caseData, hearingDynamicList, parties);
             }
 
         }
     }
 
-    private static void createDynamicJudgment(CaseData caseData, DynamicFixedListType hearingDynamicList) {
+    private static void createDynamicJudgment(CaseData caseData, DynamicFixedListType hearingDynamicList,
+                                              DynamicFixedListType partiesDynamicList) {
         var judgmentType = new JudgementType();
         judgmentType.setDynamicJudgementHearing(hearingDynamicList);
+        var reconsiderationType = new JudgmentReconsiderationType();
+        reconsiderationType.setDynamicReconsiderationPartyInitiative(partiesDynamicList);
+        judgmentType.setJudgementReconsiderations(reconsiderationType);
         var judgmentTypeItem = new JudgementTypeItem();
         judgmentTypeItem.setValue(judgmentType);
         List<JudgementTypeItem> judgementTypeList = new ArrayList<>();
         judgementTypeList.add(judgmentTypeItem);
         caseData.setJudgementCollection(judgementTypeList);
     }
 
+    private static void dynamicReconsiderations(CaseData caseData, DynamicFixedListType parties,
+                                                JudgementType judgementType) {
+        DynamicValueType dynamicValueType;
+        JudgmentReconsiderationType reconsiderationType;
+        if (judgementType.getJudgementReconsiderations() == null) {
+            reconsiderationType = new JudgmentReconsiderationType();
+            reconsiderationType.setDynamicReconsiderationPartyInitiative(parties);
+            dynamicValueType = parties.getListItems().get(0);
```

Comment:
Possibly throws an exception if the list(i.e. parties.getListItems()) is empty or null.

Reply:
parties will never be empty as it is populated with the claimant and respondents on a case. This data will always be present on a case but I can add it back in once ExUI have fixed the issue